### PR TITLE
update to v2.5.0

### LINKED ISSUE #2: Update to Parquet Format 2.5.0

### PR DESCRIPTION
**Pull request by nevi-me**
Fixes #2

### CODE REVIEW

**Comment by nevi-me** [comment]
Now that I look at the commit, it doesn't really look like there's a change between 2.4 and 2.5, other than commits on the thrift file. I think I might have done something wrong. Please don't merge this as yet, unless it's actually the case that 2.4 and 2.5 weren't different.

EDIT: I diffed the two files, looks like it's the case that the `parquet.thrift` didn't change between the two releases. https://www.diffchecker.com/5h6pDWGh

**Comment by sunchao** [comment]
Yes, there isn't any actual change on the format itself in 2.5.0, see the [release notes](https://github.com/apache/parquet-format/blob/master/CHANGES.md#version-250). Even in 2.6.0, the change is very few (nano sec in time unit and UUID type).

**Review by sunchao — APPROVED** [review]
+1. LGTM.

**Pull request merged**
